src/App.tsx

```diff
@@ -42,7 +44,7 @@ function App(): JSX.Element {
                         path={homePageRoute}
                         element={
                             <OidcSecure>
-                                <Protected allowedRole={'all'}>
+                                <Protected allowedRole={'admin'}>
```

Comment:
this is not correct. Places which had "allowedRole" as "all" before must work for uses with all 3 roles -> admin, csp and user.